contracts/package.json

```diff
@@ -15,23 +15,17 @@
     "@nomiclabs/hardhat-ethers": "^2.0.5",
     "@nomiclabs/hardhat-waffle": "^2.0.3",
     "@shardlabs/starknet-hardhat-plugin": "^0.6.2",
-    "@types/chai": "^4.2.22",
+    "@types/chai": "^4.3.3",
     "@types/elliptic": "^6.4.14",
-    "@types/mocha": "^9.0.0",
-    "@types/node": "^18.7.11",
-    "bn.js": "^5.2.0",
+    "@types/mocha": "^9.1.1",
     "cairo-ls": "^0.0.4",
-    "chai": "^4.3.4",
+    "chai": "^4.3.6",
     "ethereum-waffle": "^3.4.4",
     "ethers": "^5.6.8",
-    "hardhat": "^2.10.2",
-    "starknet": "3.9.0",
-    "ts-node": "^10.4.0",
-    "typescript": "^4.5.2"
```

Comment:
Aren't these required for tests?

Reply:
They are declared as part of `<root>/package.json` (just once, single version) and used by the whole yarn workspace. No need to redeclare them here, if a custom version is not required.